### PR TITLE
Remove method searchYAML from console-shared

### PR DESCRIPTION
**Pull request by rhrazdil**
As pointed out by @spadgett  in #2125,
searchYAML has several pitfalls and should not be used.

Removed this method from console-shared and updated test cases that used this method.

### CODE REVIEW

File: frontend/packages/kubevirt-plugin/integration-tests/tests/vm.clone.scenario.ts [hunk 1]
```diff
@@ -225,8 +225,12 @@ describe('Test clone VM.', () => {
 
     it('Cloned VM has vm.kubevirt.io/name label.', () => {
       expect(
-        searchYAML(`vm.kubevirt.io/name: ${vm.name}`, clonedVM.name, clonedVM.namespace, 'vm'),
-      ).toBeTruthy();
+        _.get(
```
Comment (spadgett) [inline frontend/packages/kubevirt-plugin/integration-tests/tests/vm.clone.scenario.ts:228]: Consider `_.has` or optional chaining here.

https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#optional-chaining

File: frontend/packages/kubevirt-plugin/integration-tests/tests/vm.clone.scenario.ts [hunk 2]
```diff
@@ -225,8 +225,12 @@ describe('Test clone VM.', () => {
...
+          getLabels(getResourceObject(vm.name, vm.namespace, 'vmi')),
+          'vm.kubevirt.io/name',
+          undefined,
```
Comment (spadgett) [inline frontend/packages/kubevirt-plugin/integration-tests/tests/vm.clone.scenario.ts:231]: No need to pass the third arg here. If no default is passed, `undefined` would be returned anyway.

File: frontend/packages/kubevirt-plugin/integration-tests/tests/vm.clone.scenario.ts (hunk 1 above)
Comment (rhrazdil) [reply frontend/packages/kubevirt-plugin/integration-tests/tests/vm.clone.scenario.ts]: Thanks, updated

File: frontend/packages/kubevirt-plugin/integration-tests/tests/vm.clone.scenario.ts (hunk 2 above)
Comment (rhrazdil) [reply frontend/packages/kubevirt-plugin/integration-tests/tests/vm.clone.scenario.ts]: I wanted to be explicit, but yeah, it's unnecessary. Removed it.

**Comment by rhrazdil** [comment]
CI keeps failing
/hold

**Pull request merged**